Sources/PostgresNIO/New/PSQLRow.swift

```diff
@@ -54,10 +54,9 @@ extension PSQLRow {
             columnIndex: index,
             file: file,
             line: line)
-        
-        guard var cellSlice = self.data[column: index] else {
-            throw PSQLCastingError.missingData(targetType: T.self, type: column.dataType, context: context)
-        }
+
+        // Safe to force unwrap here, as we have ensured above, that the row has enough columns 
```

Comment:
```suggestion
        // Safe to force unwrap here, as we have ensured above that the row has enough columns 
```
Look up "pedantic" in the dictionary, you'll see a picture of me. That picture will be placed nearby the textual definition and most likely illustrate me in the process of correcting some truly inane and unimportant grammatical error, or of significantly overexplaining a simple concept.

(So, what do you see a picture of if you look up "comedian"? 😇)

Reply:
fixed.